package.json

```diff
@@ -94,6 +98,5 @@
     "extends": [
       "@commitlint/config-conventional"
     ]
-  },
-  "packageManager": "bun@1.1.15"
```

Comment:
@gentlementlegen So what package manager should be used for the kernel, `bun` or `yarn`?

Reply:
We are still using `bun`, I'll check if I can add this back but I had issues while running Jest with this inside the `package.json`.